packages/terra-data-grid/tests/wdio/worklist-data-grid-spec.js

```diff
@@ -396,6 +396,45 @@ Terra.describeViewports('WorklistDataGrid', ['medium', 'large'], () => {
     });
   });
 
+  describe('row deletion', () => {
```

Comment:
I would think that you could test most of these scenarios with jest.  Do we really need as many WDIO tests?